test-app/src/bundles/home-page-illustration.js

```diff
@@ -12,7 +12,7 @@ function HomePageIllustration() {
       width="300"
       height="143"
       xmlns="http://www.w3.org/2000/svg"
-      xmlns:svg="http://www.w3.org/2000/svg"
+      xmlnsXlink="http://www.w3.org/2000/svg"
```

Comment:
This would be rendered as : `xmlns:xlink="http://www.w3.org/1999/xlink"` (a namespace which is not used in this file). Line 15 can simply be deleted as `http://www.w3.org/2000/svg` is already declared the default namespace in line 14. It was probably a leftover from some XML to JSX conversion.

Reply:
Of course, a copy+paste error, thanks for spotting. Will remove completely.